Assets/Scripts/ScriptableObjects/Data/HurtData.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(filename = "Hurt Data", menuname = "Hurt Data", order = 0)]
```

Comment:
Prefix menuname by the folder structure "Entity/...", like in the EffectData, ItemData or StatusEffectData scripts.

Reply:
Done. Thanks!